crates/cxx-qt-lib/src/core/qlinef.cpp

```diff
@@ -0,0 +1,20 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#include "cxx-qt-lib/qlinef.h"
+
+#include "../assertion_utils.h"
+
+#include <cstdint>
+
+// QLine has "QPoint" members - pt1, pt2
```

Comment:
this is QLineF and they are doubles not 32bit integers and the links need updating :-)

Reply:
link is ok as qline and qlineF is in same file.